index.mdx

```diff
@@ -0,0 +1,77 @@
+---
+name: Getting started
+route: /
+---
+
+# Getting Started
+
+To install the component library in your React/React Native project, run the following command:
+```
+yarn add @kiwicom/universal-components
+```
+
+## Loading the icon font
+
+For the `Icon` component to work correctly, you will need to import the `orbit-icon` font family in your project.
+
+### In a fresh `create-react-app`
+
+You could add the following snippet in `App.css`:
+```
+@font-face {
+  font-family: "orbit-icons";
+  src: url(../node_modules/@kiwicom/universal-components/lib/fonts/orbit-icons.ttf);
+}
+```
+
+>Two working examples on GitHub:    
+- [in the example folder](https://github.com/kiwicom/universal-components/tree/master/examples/create-react-app-icon-font) of the @kiwicom/universal-component repository,    
```

Comment:
Do you think that it needs an extra example? I mean, every example is definitely plus, but is it worth it in this case? If we consider that we will need to maintain and update this sub-project from time to time.
I think that implementation description itself is quite clear and simple.

Reply:
Yeah, I thought about that.
But depending on whether this repo joins Margarita or not in the future, I think people might want to look at this repo and if they see nice documentation with some examples, they might be a lot more willing to try it out and star it.
And maintaining these examples should not be a hard task (and if they ever fall out of date, people might raise an issue?)
I guess the point I was trying to make with these examples is they can clone the repo, and go into the examples folder and not touch anything else.

Reply:
You convinced me. Having repo that I can clone and start playing with right away is definitely plus.